### PR TITLE
[WIP] Added jump and fail info to pdJ

### PR DESCRIPTION
**Pull request by xarkes**
Needed for Cutter new disassembly widget.
Must be merged when complete for 3.4.0.
For https://github.com/radareorg/cutter/tree/disassemblyWidget

### CODE REVIEW

File: libr/core/disasm.c [hunk 1]
```diff
@@ -1096,6 +1096,12 @@ static void ds_begin_line(RDisasmState *ds) {
 	if (ds->pj) {
 		pj_o (ds->pj);
 		pj_kn (ds->pj, "offset", ds->vat);
+		if (ds->analop.jump != UT64_MAX ) {
+			pj_kN (ds->pj, "jump", ds->analop.jump);
```
Comment (radare) [inline libr/core/disasm.c]: why the jump is signed and the fail unsigned?

Comment (radare) [reply libr/core/disasm.c]: use pj_kn instead of pj_kN

Comment (xarkes) [reply libr/core/disasm.c]: Because it's a PoC and I don't know the API. Thanks I will fix it.

**Comment by radare** [comment]
next time push the r2r as a PR otherwise it's lying to me

**Pull request merged**